packages/ui/src/components/va-backtop/VaBacktop.stories.ts

```diff
@@ -1,12 +1,79 @@
-import { defineComponent } from 'vue'
-import VaBacktop from './VaBacktop.demo.vue'
+import { VaBacktop } from './'
+import { VaRadio } from '../va-radio'
 
 export default {
   title: 'VaBacktop',
   component: VaBacktop,
 }
 
-export const Default = defineComponent({
+export const Default = () => ({
   components: { VaBacktop },
-  template: '<VaBacktop/>',
+  template: `
+    <p class="m-1">{{$vb.lorem(10000)}}</p>
+    <va-backtop/>
+  `,
 })
+
+export const Color = () => ({
```

Comment:
I think for all stories we need some sort of "scroll into view" initial setup.

I think the preferable way to do this would be in mounted hook, but it's probably also fine to do as part of interaction script.

The problem right now is that for every story - you have to scroll manually for component to appear, otherwise you only see lorem.

This makes visual tests useless and testing cumbersome.